File: 3120-count-the-number-of-special-characters-i/3120-count-the-number-of-special-characters-i.py
class Solution(object):
    def numberOfSpecialChars(self, word):
        x=[]
        for i in word:
            x.append(i)
        y=set(x)
        z=list(y)
        a=0
        for i in z:
            if i.swapcase() in x:
                a+=1
        return a//2
        

File: 3120-count-the-number-of-special-characters-i/3120-count-the-number-of-special-characters-i.java
class Solution {
    public int numberOfSpecialChars(String word) {
        return (int) Arrays.stream(word.split(""))
                .filter(s -> Character.isLowerCase(s.charAt(0)) && word.contains(String.valueOf(Character.toUpperCase(s.charAt(0)))))
                .distinct()
                .count();
    }
}